### ISSUE TITLE
[Bug] Device_tracker

### ISSUE THREAD

**Issue by TEDSv** [tomaae/homeassistant-mikrotik_router#189]
## Describe the issue
Something wrong with device tracker with my mikrotik. I deleted integration and setup again, but device_tracker is still not working. 
All other entities don't work too, but after setup, it works (like CPU load or uptime).

## Screenshots
<details>
  <summary>Error1 in log</summary> 

![chrome_9GIDwaW3bU](https://user-images.githubusercontent.com/35523575/161998285-0a1fa123-f8a7-4aba-a279-92274c92759d.png)

</details>

<details>
  <summary>Error2 in log</summary> 

![chrome_xubqZKvnWF](https://user-images.githubusercontent.com/35523575/161998292-2842a69a-7e48-4c42-b2c3-f599c78d36f3.png)

</details>

<details>
  <summary>Configure screen</summary>

![chrome_5rtE3o1wl2](https://user-images.githubusercontent.com/35523575/161998649-29277c28-7e01-4664-a7da-4957bce9f540.png)

</details>

## Software versions
 - Home Assistant version: 2022.3.8
 - Mikrotik Router integration version: 2.0
 - Mikrotik Hardware: hAP ac^2
 - RouterOS version: v6.48.4

## Diagnostics data
[config_entry-mikrotik_router-1214557458fb113c58c44f89193cfd07.json.txt](https://github.com/tomaae/homeassistant-mikrotik_router/files/8427726/config_entry-mikrotik_router-1214557458fb113c58c44f89193cfd07.json.txt)

**Comment by tomaae** [comment]
looks like it is trying to update device that i snot registered with hass.
can you try with master? 

if it happens again, run it in [debug](https://github.com/tomaae/homeassistant-mikrotik_router#enabling-debug) until you see an error? 
I need to see everythign that happens from HA start until crash. Please include diagnostics from that run too.

**Comment by TEDSv** [comment]
Problem with Xiaomi Gateway was before the new setup. I deleted entity, restart hass and it does not help. So after that, I made a new setup.

I don't understand, what you mean about "try with master". What I should do? 

I turned to debug and restart hass, and device tracker is working now. I don't understand, it's like 10th restart for today (for mikrotik too), and somehow it works now. Let me try to figure out what happens.
